docs/content/providers/docker.md

```diff
@@ -516,7 +521,7 @@ providers:
 
 #### `tls.insecureSkipVerify`
 
-TODO add description.
+If `insecureSkipVerify` is `true`, TLS for the connection to Docker accepts any certificate presented by the server and any host name in that certificate.
```

Comment:
```suggestion
Policy for certificate verification for the secured connection to Docker.

If `insecureSkipVerify` is `true`, Traefik accepts any certificate presented by Docker and any host name in that certificate.
```

Reply:
It's not related to Traefik, but to Docker